8.4/docker-entrypoint.sh

```diff
@@ -19,6 +19,8 @@ if [ "$1" = 'postgres' ]; then
 			cat >&2 <<-'EOWARN'
 				****************************************************
 				WARNING: No password has been set for the database.
+				         This will allow anyone to connect to your
+				         postgres database.
```

Comment:
I think this needs a tiny bit more information about exactly _who_ can connect, because it's not really "anyone", it's "anyone with access to the port".

My take:
```
This will allow anyone with access to the Postgres port to access your database.  In Docker's default configuration, this is effectively any other container on the same system.

Use "-e POSTGRES_PASSWORD=password" to set a password in "docker run".
```